students/Lina/Lesson9/donor_models.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+#------------------------------------------------------
+# Lesson 9 - Assignment 8: Mailroom - Object Oriented
+#
+#            donor_models.py
+#               class Donor
+#               class DonorCollection
+#------------------------------------------------------
+
+from datetime import date
+from operator import itemgetter
+
+# Donor class contains information of a single donor
+class Donor:
+    def __init__(self, donor_name, donor_details=None):
+        self.full_name = donor_name
+        if donor_details == None:
+            details = self.lookup_donor(donor_name)
+            if details:
+                self.details = details
+            else:
+                self.details = {"gift_amounts" : [0]}
```

Comment:
why store this in a dictionary? it seems that it is always a single value.

we use classes to help us structure data, so whenever possible you want to use a class attribute to stored data.